cmd/queue/main.go

```diff
@@ -33,6 +34,8 @@ import (
 	"syscall"
 	"time"
 
+	activatorutil "github.com/knative/serving/pkg/activator/util"
```

Comment:
Retryer looks like a good candidate for https://github.com/knative/pkg.  You could just open an issue if you don't want to move it now.

Reply:
@markusthoemmes I wonder if https://godoc.org/k8s.io/apimachinery/pkg/util/wait would work for this case. Then we wouldn't have to maintain our own backoff code.

Reply:
Oh good one @grantr, I'll take a look at those and see if we can replace accordingly!